src/comparsion.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+pd.set_option('display.max_columns', None)
+pd.set_option('display.width', 1000)
+
+import json
+import re
+from pathlib import Path
+
+# orginally clojure_mapping
+combined_mapping = {
+    'pre-increment': 'preIncr',
+    'operator-precedence': 'operator_precedence',
+    'conditional': 'conditional',
+    'preprocessor-in-statement': 'preprocessor_in_statement',
+    'logic-as-control-flow': 'logic_as_control_flow',
+    'type-conversion': 'type_conversion',
+    'comma-operator': 'comma_atoms',
+    'implicit-predicate': 'implicit_predicate',
+    'post-increment': 'postIncr',
+    'assignment-as-value': 'assignment_as_value',
+    'repurposed-variable': 'repurposed_variable',
+    'omitted-curly-braces' : 'omitted_curly_braces',
+    'macro-operator-precedence' : 'macro_operator_precedence',
+    'literal-encoding' : 'literal_encoding'
+}
+
+CLJnCQL = ["CLJ", "CQL"]
+CLJnCNL = ["CLJ", "CNL"]
+CQLnCNL = ["CQL", "CNL"]
+all = ["CLJ", "CQL", "CNL"]
```

Comment:
Simple lists that helps me reindex columns later. Each list contains exactly the names of the two or all options.